apps/web/components/molecules/userAvatar/userAvatar.tsx

```diff
@@ -3,11 +3,16 @@ import clsx from 'clsx';
 
 type CommentUserAvatarProps = {
 	avatar: string;
-	size?: number;
+	size?: '12' | '10' | '24';
 };
 
-export const UserAvatar = ({ avatar, size = 24 }: CommentUserAvatarProps) => {
-	const sizeClass = `w-${size} h-${size}`;
+export const UserAvatar = ({ avatar, size = '24' }: CommentUserAvatarProps) => {
+	const sizes = {
+		24: 'w-24 h-24',
+		12: 'w-12 h-12',
+		10: 'w-10 h-10',
+	};
```

Comment:
This can be placed outside of the component as a const

`SIZES` or `AVATAR_SIZES`